### PR TITLE
Merge config

### PR DESCRIPTION
**Pull request by smacker**
Fix: #20

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,6 +47,37 @@ Any of the analyzers or a default dummy one, included in this repository
...
+```yml
+analyzers:
+  - name: Example name # required, unique name of the analyzer
+    addr: ipv4://localhost:10302 # required, gRPC address
+    disabled: false # optional, false by default
+    settings: # optional, this field is sent to analyzer "as is"
+        threshold: 0.8
+```
+
+It's possible to override configuration for a repository.
```
Comment (bzz) [inline README.md]: It's possible to override Analyzers configuration for a particular repository.

File: README.md [hunk 2]
```diff
@@ -47,6 +47,37 @@ Any of the analyzers or a default dummy one, included in this repository
...
+To do that `.lookout.yml` must be present.
```
Comment (bzz) [inline README.md]: To do that `.lookout.yml` must be present in the root of that repository.

File: README.md [hunk 3]
```diff
@@ -47,6 +47,37 @@ Any of the analyzers or a default dummy one, included in this repository
 1. Create a new pull requires in the repository
 
 
+# Configuration file
+
+Server configuration `config.yml`:
```
Comment (bzz) [inline README.md]: > Global server configuration is stored in `config.yml`

We use word global later, so it's nice to introduce it early in here

**Comment by smacker** [comment]
Thank you for the suggestions! Fixed.

I thought to add some kind of integration tests but as you mentioned it might be out of the scope of this PR and I also think it's a little bit premature and would slow down CI.
There are unit tests for enabled/disabled analyzers already. I hope it's enough for now. We can improve testing later.

**Pull request merged**